docs/notebooks/geolift1.ipynb

```diff
@@ -0,0 +1,614 @@
+{
```

Comment:
<p>turn progress bar off.</p><p> </p><p>Also, briefly explain what this is doing, maybe with some references to other docs for more info.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/CausalPy/pull/135/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Done</p>